### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by jihoonyyy**
Inserted the table into "more detail" and the table has red and green color for closed and open.

### CODE REVIEW

File: free_court/src/components/CalendarBox/CalendarBox.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```
Comment (Mark-Jung) [inline free_court/src/components/CalendarBox/CalendarBox.js]: What is this?

**Review by Mark-Jung — CHANGES_REQUESTED** [review]
One unnecessary component

File: free_court/src/components/Calendar/Calendar.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { CalendarBox } from '../CalendarBox';
+
+
+
+class CalComponent extends Component {
+
+  createTable = () => {
+      let table = []
+      // Outer loop to create parent
+      for (let i = 5; i < 25; i++) {
```
Comment (Mark-Jung) [inline free_court/src/components/Calendar/Calendar.js]: This is dependent on the hours showing 5 to 25. Make this component independent on how many hours are required for showing.

File: free_court/src/components/CalendarBox/CalendarBox.js (hunk 1 above)
Comment (jihoonyyy) [reply free_court/src/components/CalendarBox/CalendarBox.js]: Deleted it

File: free_court/src/components/Calendar/Calendar.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+  render() {
+    return (
+      <div className="center">
+        <table style={{
+          width:"500"
+          // minWidth: "550px",
+          // borderLeft: "solid",
+          // borderColor: "RGBa(61, 61, 61,.3)",
```
Comment (Mark-Jung) [inline free_court/src/components/Calendar/Calendar.js]: delete unnecessary comments

**Review by Mark-Jung — CHANGES_REQUESTED** [review]
foreach!

**Pull request merged**